src/Configuration/Provisioning/data/utils.js

```diff
@@ -1,13 +1,26 @@
 import { useContextSelector } from 'use-context-selector';
 import { ProvisioningContext } from '../ProvisioningContext';
 
+// Takes an array of catalog queries and sorted them by last modified date (newest first)
+export function sortedCatalogQueries(catalogQueries) {
+  return catalogQueries.sort((b, a) => {
+    if (a.modified < b.modified) {
+      return -1;
+    }
+    if (a.modified > b.modified) {
+      return 1;
+    }
+    return 0;
+  });
+}
```

Comment:
nit: The javascript sort function only counts positive and negative numbers (besides zero), they don't have to be +/- 1.
```suggestion
export function sortedCatalogQueries(catalogQueries) {
  return catalogQueries.sort((b, a) => a - b);
}
```

Reply:
Hey Marlon, I tried refactoring the code, but it would not sort the dates in  the correct order. Doing a greater than less then on the value and sorting that way seems to be working correctly though and is passing test / live sanity check